app/src/app/components/contents/explorer/PacketGroupSummaryList.tsx

```diff
@@ -0,0 +1,75 @@
+import { Dispatch, SetStateAction } from "react";
+import useSWR from "swr";
+import appConfig from "../../../../config/appConfig";
+import { fetcher } from "../../../../lib/fetch";
+import { PageablePacketGroupSummary } from "../../../../types";
+import { Skeleton } from "../../Base/Skeleton";
+import { Pagination } from "../common/Pagination";
+import { PacketGroupSummaryListItem } from "./PacketGroupSummaryListItem";
+
+interface PacketGroupSummaryListProps {
+  filterByName: string;
+  pageNumber: number;
+  pageSize: number;
+  setPageNumber: Dispatch<SetStateAction<number>>;
+}
+export const PacketGroupSummaryList = ({
+  filterByName,
+  pageNumber,
+  pageSize,
+  setPageNumber
+}: PacketGroupSummaryListProps) => {
+  const { data, isLoading, error } = useSWR<PageablePacketGroupSummary>(
+    `${appConfig.apiUrl()}/packets/packetGroupSummary?pageNumber=${pageNumber}&pageSize=${pageSize}\
+    &filterName=${filterByName}`,
+    (url: string) => fetcher({ url, authRequired: true })
```

Comment:
I just realised through doing the download ticket that you'll probably need to set `authRequired`here to false if running with auth not enabled.

Reply:
as discussed have created a new ticket for this and will be sorted as part of that

Reply:
Adding link for findability: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4771/Fronted-Adjust-layouts-fetching-to-account-for-non-auth-mode